### ISSUE TITLE
Optimizers not using specified arguments in the Transformers integration

### ISSUE THREAD

**Issue by eldarkurtic** [neuralmagic/sparseml#217]
**Describe the bug**
Optimizers in `integrations/transformers` are always initialized with default arguments. Any user-defined arguments (for example via command line and argparser) are completely discarded.

**Expected behavior**
To be able to set optimizer's arguments, for example learning rate.

**To Reproduce**
Run any `integrations/transformers` example python script and compare specified optimizer's params and the actual optimizer's params (for example `--learning_rate X`).
